FVCalc.java

```diff
@@ -1,6 +1,26 @@
 // Computes the future value of a saving investment.
 public class FVCalc {
 	public static void main(String[] args){
```

Comment:
Variables like currentvalue, futurevalue, yearfirst, and Orate don't follow standard Java naming conventions. Use camelCase for readability and meaningful names.
Should be - 
currentValue 
futureValue
yearFirst
oRate